includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -1986,6 +1988,35 @@ public function is_in_test_mode() {
 		return WC_Payments::get_gateway()->is_in_test_mode();
 	}
 
+	/**
+	 * Send WC pay request using value objects.
+	 *
+	 * @param WC_Pay_Request $request - Request object.
+	 *
+	 * @return Response
+	 * @throws InvalidArgumentException|API_Exception
+	 */
+	public function send_wcpay_request( WC_Pay_Request $request ) {
+		$params = $request->get_request_data();
+
+		// If level3 data doesn't contain any items, add a zero priced fee to meet Stripe's requirement.
+		if ( ! isset( $params['level3']['line_items'] ) || ! is_array( $params['level3']['line_items'] ) || 0 === count( $params['level3']['line_items'] ) ) {
```

Comment:
I will keep it inside `get_request_data()` since keeping it in trait is great if this code will be used somewhere else, since we are not planning to use this logic somewhere else in core (for now).

Reply:
Addressed in https://github.com/Automattic/woocommerce-payments/pull/5012/commits/26096e9af2b9689a60d1d7b71ae9fac53c441b97